rasa/test.py

```diff
@@ -27,6 +32,20 @@ def test_core_models_in_directory(model_directory: Text, stories: Text, output:
     plot_core_results(output, number_of_stories)
 
 
+def _test_core_models_in_directory_input(model_directory: Text) -> Text:
+    import rasa.model
+
+    p = Path(model_directory)
+    if p.is_file():
+        if model_directory != rasa.model.get_latest_model():
```

Comment:
I'd add a comment why we are checking the model_directory against the `get_latest_model`. This check might otherwise be a bit confusing for somebody coming this code and not knowing the context. E.g.
```suggestion
        # If the passed model is the latest model, it's probably caused by the default value of the CLI. Don't warn in this case."
        if model_directory != rasa.model.get_latest_model():
```

Reply:
alternatively you could write a docstring for `_test_core_models_in_directory_input` and describe it's functionality in there (probably the better way than the comment)